projects.json

```diff
@@ -1,10 +1,11 @@
 [
     {
         "id": 1,
-        "name": "Création de ce site web",
+        "name": "Website creation ",
         "mainImg":"/media/vue-logo-1.png",
         "link": "/",
-        "description":"Mon portfolio WordPress n'étant pas à mon goût, j'ai décidé de faire ce site entièrement à la main et d'apprendre VueJS au passage, grâce à la documentation et à beaucoup de recherches sur Google. Toutes les animations de ce site sont en CSS et facilitées par la balise 'transition' de Vue. Le seul Javascript (hors VueJS) présent est utilisé pour les changement de page.",
+        "description":"My Wordpress portfolio wasn't to my liking thus i decided to create this website from scratch and to learn VueJS on the way, thanks to the online documentation and a lot of Google searches. All of this website's animations are done in CSS and helped by the 'transition' tag from Vue. The only JS used here (except Vue) is for the change of pages ",
```

Comment:
thus I (i capitalization) 
thanks to it's documentation and a lot of Google searches
is for the page's navigation, instead of "for the change of pages"

Reply:
its*